### DISCUSSION TITLE
Can't deploy on github Pages

### DISCUSSION THREAD

**Discussion by FabricioNarvaez** [community/community#51387] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've tried to deploy this repository on gitHub Pages (https://github.com/AgrupacionPaternain/agrupacionpaternain.github.io) but I can't.
This are the settings. 
![image](https://user-images.githubusercontent.com/129317005/228623905-42564e55-a6e9-4c75-aabc-1223f5edb3be.png)
It's a simple web with html, css and js. On gh-pages branch I have index.html and assets folder with css ans js.
 On actions sections I have this, don't know if it's the problem.
![image](https://user-images.githubusercontent.com/129317005/228623553-a0a3e9f4-d0e8-410d-93fc-fe04a3250c3a.png)

**Comment by CodeMaster7000** [comment, marked as answer]
Hi @FabricioNarvaez,

This looks like a temporary GitHub servers problem. This is the case as GitHub Pages are running at "degraded" performance. This issue has been reported [here](https://www.githubstatus.com/incidents/z3c6q056q332) and is under investigation.

You can check the [GitHub status page](https://www.githubstatus.com) to get real-time information on current performance. There is also the option of subscribing for updates.
